app/views/success/edit.html.erb

```diff
@@ -3,9 +3,22 @@
 <div class="form-card">
   <%= form_for @step, as: :step, builder: MbFormBuilder, url: current_path, method: :put do |f| %>
     <header class="form-card__header">
+      <div class="illustration illustration--success"></div>
+      <div class="pre-title">Congratulations</div>
       <div class="form-card__title">
-        Your Michigan Benefits application is ready to be submitted!
+        Your application has been successfully submitted.
       </div>
+      <div class="text--help text--centered">
+        Submitted to MDHHS on <%= @step.submitted_date %>.
```

Comment:
when this is nil, it will say `Submitted to MDHHS on .` is that OK?

Reply:
In practice someone should never see this page until they've signed. I added the nil check so that us developers don't get an error when skipping steps :)